tests/evaluation_metrics/conftest.py

```diff
@@ -4,7 +4,7 @@
 
 
 @pytest.fixture
-def get_gt_pred_bbox():
+def get_mini_batches():
```

Comment:
Why not create another fixture so that you don't have to change `test_confusion_metrix.py`?